ui/raidboss/data/05-shb/raid/e3s.js

```diff
@@ -115,8 +115,8 @@
       alertText: function(data, matches) {
         if (data.me == matches.target) {
           return {
-            en: 'Tank Buster on YOU',
-            de: 'Tankbuster auf DIR',
+            en: 'Tank buster on YOU',
```

Comment:
The previous was fine.  I'm not sure why you are changing this.

Reply:
Oh , sorry.